src/services/to-image.service.ts

```diff
@@ -1,37 +1,35 @@
-import { toBlob } from 'html-to-image';
 import { saveAs } from 'file-saver';
+import html2canvas from 'html2canvas';
 
 const DEFAULT_SCALE = 1;
 
-const widgetToImage = (fileName: string, widgetElement: HTMLElement, scale = DEFAULT_SCALE) => {
-  usingHtmlToImage(fileName, widgetElement, scale);
+const removeMapControllers = (el: Document) => {
+  const elementList = el.querySelectorAll(
+    '.leaflet-control-container, .gmnoprint, .gm-fullscreen-control, .gm-iv-address, .gm-style-cc',
+  );
+  Array.from(elementList).forEach((el) => el.remove());
 };
 
-// Uses SVG foreignObject. Works with canvas and HTML/SVG
-// https://github.com/bubkoo/html-to-image#how-it-works
-const usingHtmlToImage = (fileName: string, widgetElement: HTMLElement, scale: number) => {
-  const leafletControls = [
-    'leaflet-control-container',
-    'gmnoprint',
-    'gm-fullscreen-control',
-    'gm-iv-address',
-    'gm-style-cc',
-  ];
+const widgetToImage = (fileName: string, widgetElement: HTMLElement, scale = DEFAULT_SCALE) => {
+  usingHtml2Canvas(fileName, widgetElement, scale);
```

Comment:
This will us only `html2canvas` and never `html-to-image`
as far as I know, `html2canvas` works OK for maps, but not for other elements lie pie charts(we used to have some RTL issues)

If this is still the case - use similar logic as we did before (see https://github.com/hasadna/anyway-newsflash-infographics/blob/c0041dd29a21a704f9dc192a2e2bc9a94131da50/src/services/to-image.service.ts)

If can work for all cases - please remove `html-to-image` to image package (please check it intensively before you do)